runtime/parachains/src/runtime_api_impl/v3.rs

```diff
@@ -11,22 +11,25 @@
 // MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
```

Comment:
We still have a reference to v2 on line 93, is this correct?

```rs
target: "runtime::polkadot-api::v2",
```

Reply:
Good catch, ty!